dev/com.ibm.ws.anno/src/com/ibm/ws/annocache/targets/cache/internal/TargetCacheImpl_Reader.java

```diff
@@ -383,16 +481,20 @@ public void handleTrailer() {
     }
 
     public class ContainerTableReader extends TargetsReader {
-        public ContainerTableReader(TargetsTableContainers containerTable) {
+        public ContainerTableReader(TargetsTableContainers containerTable, TargetCacheImpl_Reader reader) {
```

Comment:
since it is an inner class it has a pointer to its outer class.  If it doesn't use it, should the class be made static now that the Reader is passed into the constructor?